app/src/main/kotlin/org/fossify/messages/extensions/Context.kt

```diff
@@ -365,7 +329,15 @@ fun Context.getConversations(
             }
 
             // drafts are stored locally they take priority over the original date
-            val draft = draftsDB.getDraftById(id)
+            val draft = try {
+                draftsDB.getDraftById(id)
+            } catch (e: Exception) {
+                if (e.message == "Cannot access database on the main thread since it may potentially lock the UI for a long period of time.") {
+                    null
+                } else {
+                    throw e
+                }
+            }
```

Comment:
This code is always supposed to be called in a background thread.

Also, that's not a good way to catch an exception.